components/CardAuthentication/src/story-template.tsx

```diff
@@ -103,27 +103,29 @@ export const AuthenticationCard = ({
         )}
       </header>
       <div className="denhaag-card-authentication__cta">
-        <a
-          className="denhaag-card-authentication__cta-link denhaag-button denhaag-button--large denhaag-button--primary-action denhaag-button--end-icon"
-          href="#example-link"
-          target="_self"
-          title={title}
-        >
-          Inloggen
-          <span className="denhaag-button__icon">
-            <svg
-              aria-hidden="true"
-              className="denhaag-icon"
-              focusable="false"
-              height="1em"
-              viewBox="0 0 24 24"
-              width="1em"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z" />
-            </svg>
-          </span>
-        </a>
+        <div className="denhaag-button-group denhaag-button-group--single">
```

Comment:
I think the `denhaag-button-group--single` doesn't exist anymore?

Reply:
I've removed all `denhaag-button-group--single` and `denhaag-button-group--multiple` classes, since we don't use them anymore.